### ISSUE TITLE
Немного изменить верстку рекомендуемых статей на узких экранах

### ISSUE THREAD

**Issue by akellbl4** [web-standards-ru/web-standards.ru#139]
Собственно говоря пример на изображении ниже: сверху – как сейчас, снизу – как предлагается.
В макетах с новым дизайном еще нет статьи на разных разрешениях поэтому импровизировал как мог (шрифт в моем варианте Helvetica, просто за не имением используемого :)

![CleanShot 2020-10-18 at 22 46 16](https://user-images.githubusercontent.com/2330682/96378362-17610b80-1194-11eb-9b93-14a025b6d04d.png)

**Comment by pepelsbey** [comment]
Я бы всё же нарисовал карточки, а потом уже верстал бы их. Сейчас уже на главной есть узкая карточка, но не факт, что она так же будет выглядеть.

<img width="305" alt="image" src="https://user-images.githubusercontent.com/105274/96385047-f26f9700-1199-11eb-8f0a-d47b3937f463.png">

**Comment by akellbl4** [comment]
Взял карточку из нового дизайна
![CleanShot 2020-10-19 at 11 26 30](https://user-images.githubusercontent.com/2330682/96420697-7b201e80-11fe-11eb-8ebe-a92f08f4b245.png)

**Comment by pepelsbey** [comment]
Тут задача чуть сложнее, чем просто сделать получше. Нужно взять блок карточки и переверстать его во всех видах. Чтобы она адаптировалась и компактно, и широко, и средне. См. новый дизайн карточки на главной и в списке статей.

**Comment by akellbl4** [comment]
Да, я понимаю. Последний пример как раз взят от туда. Но идею я понял, не просто поправить, а прям переверстать ;)

**Comment by higimo** [comment]
@pepelsbey можешь обозначить статус: сейчас нарисованы версии, если нет, то какие и какие к ним могут быть замечания?
